### ISSUE TITLE
Dataset for cross domain experiment.

### ISSUE THREAD

**Issue by by5747** [rajeevyasarla/Syn2Real#11]
Hi, I am curious about the experimental dataset.

You used 9,100 train images in DDN when you did real experiments. 
Did you use the same 9,100 sheets for cross domain experiment? Or did you use 12,600 sheets as an official dataset?
I wonder if the Rain200H and Rain800 also used all of the training sets for cross domain experiment.

**Comment by rajeevyasarla** [comment]
For cross domain experiments I used DIDMDN as source or labeled dataset, and DDN, Rain200L and Rain800 as target or unlabeled dataset.
DDN dataset contains 14000 images where 12600 are training and 1400 images testing. So among 12600 training images I used only 9100 (indexing 1 - 650 i.e each clean image has 14 different rain images). And for testing I used 1400 images from the DDN test set.

For example
So for DIDMDN to DDN cross domain experiment. DIDMDN is label and DDN is unlabeled.
So for DIDMDN to Rain200L cross domain experiment. DIDMDN is label and Rain200L is unlabeled.
So for DIDMDN to Rain800 cross domain experiment. DIDMDN is label and Rain800 is unlabeled.
